scripts/wasm-rebuild/docker-scripts/rebuild_current.sh

```diff
@@ -41,7 +41,14 @@ if [ -d jsoncpp ]; then
 fi
 
 set +e
-scripts/*/build_emscripten.sh
+
+if [ -e scripts/c1/build_emscripten.sh ]; then
```

Comment:
```suggestion
if [ -e scripts/ci/build_emscripten.sh ]; then
```

Reply:
Thanks. I thought I fixed it but turns out that only on the line below :(